utils/docker-compose/envoy.yaml

```diff
@@ -50,138 +50,180 @@ static_resources:
-                              "upstream_cluster": "limitador",
+                              "name": "rlp-ns-A/rlp-name-A",
+                              "domain": "rlp-ns-A/rlp-name-A",
+                              "service": "limitador",
                               "hostnames": ["*.a.com"],
-                              "gateway_actions": [
+                              "rules": [
                                 {
-                                  "configurations": [
+                                  "data": [
                                     {
-                                      "actions": [
-                                        {
-                                          "request_headers": {
-                                            "header_name": "not-existing-header",
-                                            "skip_if_absent": true,
-                                            "descriptor_key": "my_not_existing_header"
-                                          }
-                                        }
-                                      ]
+                                      "selector": {
+                                        "selector": "unknown.path"
+                                      }
                                     }
                                   ]
                                 }
                               ]
                             },
                             {
-                              "name": "rlp-b",
-                              "rate_limit_domain": "rls-domain-b",
-                              "upstream_cluster": "limitador",
+                              "name": "rlp-ns-B/rlp-name-B",
+                              "domain": "rlp-ns-B/rlp-name-B",
+                              "service": "limitador",
                               "hostnames": ["*.b.com"],
-                              "gateway_actions": [
+                              "rules": [
                                 {
-                                  "rules": [
-                                    {
-                                      "paths": ["/not/exising/path", "/any*"]
-                                    }
-                                  ],
-                                  "configurations": [
+                                  "conditions": [
                                     {
-                                      "actions": [
+                                      "allOf": [
                                         {
-                                          "generic_key": {
-                                            "descriptor_key": "some_generic_key",
-                                            "descriptor_value": "some_generic_value"
-                                          }
+                                          "selector": "request.url_path",
+                                          "operator": "starts_with",
```

Comment:
RLPv2 [specifies](https://github.com/Kuadrant/kuadrant-operator/blob/529ae66e7514d5d28df76b1d039ac21136a1babf/api/v1beta2/ratelimitpolicy_types.go#L48) and effectively the Kuadrant Operator [generates](https://github.com/Kuadrant/kuadrant-operator/blob/529ae66e7514d5d28df76b1d039ac21136a1babf/pkg/rlptools/wasm/types.go#L15) for the WasmPlugin config `startswith`, not `starts_with`.

In fact, while testing this branch, I've observed the following error thrown by the gateway:

```
2023-07-25T12:20:20.200842Z     warning envoy wasm external/envoy/source/extensions/common/wasm/context.cc:1151 wasm log: failed to parse plugin config: unknown variant `startswith`, expected one of `eq`, `neq`, `starts_with`, `ends_with`, `matches` at line 1 column 184     thread=20
```